includes/frontend.php

```diff
@@ -344,6 +345,45 @@ public function remove_content_filter() {
 		remove_filter( 'the_content', [ $this, 'apply_builder_in_content' ], self::THE_CONTENT_FILTER_PRIORITY );
 	}
 
+	/**
+	 * Registers script modules.
+	 *
+	 * Registers all the frontend script modules.
+	 *
+	 * Fired by `wp_enqueue_scripts` action.
+	 *
+	 * @since 3.28.0
+	 * @access public
+	 */
+	public function register_script_modules(): void {
```

Comment:
@rami-elementor Yes, I believe it is possible to merge this logic to the register_scripts() method.

I will leave it for me. I would like to make changes only, once we have Yakir's feedback as well.